requirements-python.txt

```diff
@@ -2,7 +2,8 @@ PyYAML==5.1
 paramiko==2.7.1
 fabric==2.5.0
 invoke==1.3.1
-boto3==1.9.200
+boto3==1.13.5
```

Comment:
Did you run some tests to make sure it didn't break anything?

Reply:
unit test was running, and seem that it didn't broke them
they are using only the s3 parts.

Reply:
now I've tagged this PR with `test-provision-aws`, let's see how it would work:
https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-2126/2/